torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -1165,14 +1166,18 @@ def elastic_image_tensor(
         return image
 
     shape = image.shape
+    device = image.device
 
     if image.ndim > 4:
         image = image.reshape((-1,) + shape[-3:])
         needs_unsquash = True
     else:
         needs_unsquash = False
 
-    output = _FT.elastic_transform(image, displacement, interpolation=interpolation.value, fill=fill)
+    image_height, image_width = shape[-2:]
+    identity_grid = _create_identity_grid((image_height, image_width), device=device)
+    grid = identity_grid + displacement.to(device)
```

Comment:
```suggestion
    grid = _create_identity_grid((image_height, image_width), device=device).add_(displacement.to(device))
```

or won't this work due to dtype differences?

Reply:
Good suggestion! Let me check that.